CONTRIBUTING.md

```diff
@@ -619,6 +626,95 @@ src/
 describe('Checkbox Basics', () => {});
 ```
 
+##### `*.test.aria.ts`
+
+Assertions against a component's accessibility tree.
+
+##### `*.test.basics.ts`
+
+A bit of a misnomer, but we've yet to think of a better name.
+These are assertions against a component in its initial state—even if it's not its default slot.
+Think of attributes and slots provided on first render.
+
+##### `*.test.events.ts`
+
+Tests that exclusively assert that a certain event has been dispatched.
+
+Don't forget to assert the event's `composed` and `bubbles` properties and, if appropriate, the event's `target`.
+It's also good to assert, using `sinon`, that an event was only dispatchd once.
+
+##### `*.test.focus.ts`
+
+Tests that exclusively assert that an element is focused after some interaction.
+Each test should mainly be asserting against `document.activeElement`.
+
+##### `*.test.interactions.ts`
+
+A dumping ground for most interaction assertions.
+An interaction can be a simulated user interaction such as a click.
+Or it can be a developer interaction—such as programmatically setting an attribute or changing a slot after initial render.
+
+This suite can get pretty large.
+So we break down specific types of interactions into two sub-suites: `*.test.events.ts` and `*.test.focus.ts`.
```

Comment:
It would make for a long file name. But `*.test.interactions.events.ts` and `*.test.interactions.focus.ts` would probably make things clearer. What do you think?

Reply:
I'd be up for it.